demo/admin/src/products/ProductForm.cometGen.ts

```diff
@@ -0,0 +1,68 @@
+import { FormConfig } from "@src/generator/generator";
+import { GQLProduct } from "@src/graphql.generated";
+
+export const ProductForm: FormConfig<GQLProduct> = {
+    type: "form",
+    gqlType: "Product",
+    fragmentName: "ProductFormDetails", // configurable as it must be unique across project
+    fields: [
+        {
+            type: "text",
+            name: "title",
+            label: "Titel", // default is generated from name (camelCaseToHumanReadable)
+            required: true, // default is inferred from gql schema
+        },
+        { type: "text", name: "slug" },
+        { type: "text", name: "description", label: "Description", multiline: true },
```

Comment:
Props like `multiline`, `required`, `validate` etc. could be placed in a `fieldProps` option

Reply:
why add this extra nesting level?

Reply:
My intension was adding a level that's passed directly to the field without the generator knowing anything about it. Or should the generator be strict about which props can be set for field?

Reply:
I see the generator as abstraction layer with a simplified api. it also uses the graphql schema and mixes that into the config. That's why I would leave it like that.

Maybe we will find out later that many props are only passed and revisit that idea.